loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
 		link_dp = path.dirname(frum)
 		if not path.exists(link_dp):
 			makedirs(link_dp)
 		if path.lexists(frum): # shouldn't be the case, but helps debugging
```

Comment:
I'm happy to merge your PR. I wish I knew what was causing it to happen, but better safe :smile: